language/vm/src/file_format.rs

```diff
@@ -390,34 +435,34 @@ pub struct FunctionSignature {
         any(test, feature = "fuzzing"),
         proptest(strategy = "vec(any::<SignatureToken>(), 0..=params)")
     )]
-    pub return_types: Vec<SignatureToken>,
+    pub return_: Vec<SignatureToken>,
     /// The list of arguments to the function.
     #[cfg_attr(
         any(test, feature = "fuzzing"),
         proptest(strategy = "vec(any::<SignatureToken>(), 0..=params)")
     )]
-    pub arg_types: Vec<SignatureToken>,
+    pub parameters: Vec<SignatureToken>,
     /// The type formals (identified by their index into the vec) and their kind constraints
-    pub type_formals: Vec<Kind>,
+    pub type_parameters: Vec<Kind>,
 }
```

Comment:
:) yes but it's still used by the compiler and I should have moved it there. 
I did not because again I want to get this in a different clean up and given we may make more changes to the separation of `CompiledModule` and such.
Bottom line I'll change it very very soon, but not in this PR